packages/sdk/src/client-v2.ts

```diff
@@ -184,13 +185,26 @@ export const createTownsClient = async (
         }
     }
 
-    const getStream = async (streamId: string): Promise<ParsedStreamResponse> => {
+    const getStream = async (streamId: string): Promise<StreamStateView> => {
         const { disableHashValidation, disableSignatureValidation } = client
         const stream = await client.rpc.getStream({ streamId: streamIdAsBytes(streamId) })
-        return unpackStream(stream.stream, {
+        const unpackedResponse = await unpackStream(stream.stream, {
             disableHashValidation,
             disableSignatureValidation,
         })
+        const streamView = new StreamStateView(userId, streamIdAsString(streamId), undefined)
+        streamView.initialize(
```

Comment:
we're creating a new stream view per `getStream(sameStream)` request
can we reuse it? (if defined)